src/rank_llm/rerank/reranker.py

```diff
@@ -100,8 +103,8 @@ def write_rerank_results(
         top_k_candidates: int = 100,
         dataset_name: str = None,
         rerank_results_dirname: str = "rerank_results",
+        ranking_execution_summary_dirname: str = "ranking_execution_summary",
         inference_invocations_history_dirname: str = "inference_invocations_history",
```

Comment:
ranking_execution_summary_dirname is replaced by inference_invocations_history_dirname, bad rebase?